fashion-example-query/README.md

```diff
@@ -5,81 +5,101 @@
 - [Run the Hello World example using query language](#run-the-hello-world-example-using-query-language)
   - [🗝️ Key Concepts](#️-key-concepts)
   - [🏃 Run the Flows](#-run-the-flows)
-  - [Results](#results)
-  - [QueryLanguage](#querylanguage)
+  - [🌟 Results](#results)
+  - [🧞‍QueryLanguage](#querylanguage)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Run the Hello World example using query language
+# Run Fashion-MNIST Example Using Query Language
 
-## 🗝️ Key Concepts
+This is an extension of the Hello World example using QueryLanguage. 
+We will use the FashionMNIST dataset to index and query. The dataset has different categories such as t-shirts, trousers, and pullovers. We will use QueryLanguage to create a different index per each category so we can query through them. 
 
-First of all, read up on [Jina 101](http://101.jina.ai) so you have a clear understanding of how Jina works. We're going to refer to those concepts a lot. We assume you already have some knowledge of Python and machine learning.
 
-This is an extension of the Hello World example, but here we use QueryLanguage to create an index per category and then, with that we can filter by category
+## 🗝️ Expected knowledge 
 
+1. Read up on [Jina 101](http://101.jina.ai). 
+2. Python knowledge.
+3. [Hello World Example](https://docs.jina.ai/chapters/helloworld/index.html) understanding.
```

Comment:
You have already run the [Hello World Example](https://docs.jina.ai/chapters/helloworld/index.html) successfully.